snuba/utils/streams/kafka.py

```diff
@@ -38,6 +38,13 @@
 from snuba.utils.streams.producer import Producer
 from snuba.utils.streams.types import Message, Partition, Topic
 
+try:
+    # PickleBuffer is only available in Python 3.8 and above and when using the
+    # pickle protocol version 5 and greater.
+    from pickle import PickleBuffer
+except ImportError:
+    pass
```

Comment:
We're still running tests against 3.7, as soon as we stop doing that this can be cleaned up (as well as the xfail in the tests.)